flake.nix

```diff
@@ -0,0 +1,57 @@
+{
```

Comment:
Could we split the flake changes into a separate PR?

Reply:
Yeah, had it lying around and got lazy. Will separate it. :)

Reply:
Well and its was in a different commit, so it felt OK to me. But I have no problem creating another PR.